docs/run-prebuilt-wda.md

```diff
@@ -77,3 +77,9 @@ We haven't tested all possible combinations, but probably the target device coul
 
 The same thing could achieve with `derivedDataPath` and `usePrebuiltWDA` capabilities, but it may fail if the `xcodebuild` cannot find or handle the `.xctestrun` properly.
 The stability depends on Xcode.
+
+## Download prebuilt WebDriverAgent from GitHub appium/WebDriverAgent repository
+
+Still preparing GitHub Actions to upload these prebuilt WebDriverAgent packages for simulators to GitHub releases, but currently you can find these packages in [Building WebDriverAgent](https://github.com/appium/WebDriverAgent/actions/workflows/wda-package.yml) workflow.
```

Comment:
lets add that after the WDA pipeline is merged

Reply:
The WDA workflow is now in the master. It will work every release at least. The upload destination is the workflow result for now.

Reply:
@KazuCocoa do you want to update the "Still preparing GitHub Actions" statement or merge it as is?

Reply:
oh, forgot this release task. They are not in the GH release page yet https://github.com/appium/WebDriverAgent/releases , so here should be as-is yet.

I'll work on making these packages downloadable from GH release page in addition to existing assets